op-chain-ops/genesis/config.go

```diff
@@ -217,13 +215,19 @@ type DeployConfig struct {
 	SuperchainConfigInitiator common.Address `json:"superchainConfigInitiator"`
 	// SuperchainConfigVetoer represents the address of the SuperchainConfigVetoer role.
 	SuperchainConfigVetoer common.Address `json:"superchainConfigVetoer"`
+	// SuperchainConfigGuardian represents the GUARDIAN account in the SuperchainConfig. Has the ability to pause withdrawals.
+	SuperchainConfigGuardian common.Address `json:"superchainConfigGuardian"`
 	// SuperchainConfigDelay represents the time after which an upgrade can occur ifit
 	SuperchainConfigDelay uint64 `json:"superchainConfigDelay"`
 	// SuperchainConfigMaxPause represents the time after which an upgrade can occur ifit
 	SuperchainConfigMaxPause uint64 `json:"superchainConfigMaxPause"`
 	// SequencerKeys represents the set of allowed Sequencer key pairs (batcher hash and unsafe
 	// block signer).
 	SuperchainConfigSequencerKeyPairs []SequencerKeyPair `json:"superchainConfigSequencerKeyPairs"`
+	// Legacy config values.
+	// These are no longer used, but are necessary for parsing legacy deploy-configs
+	PortalGuardian   common.Address `json:"portalGuardian"`
```

Comment:
We will want to remove these longer term

Reply:
Agreed... I did not feel good about this, but I didn't really see another way available here.

Reply:
We should just delete these imo and bite the bullet

Reply:
We agree to live with this for the moment.